frontend/src/mixins/GroupGuardMixin.js

```diff
@@ -1,8 +1,8 @@
 import store from '../store';
-import { groupBasedRoutes } from '../router';
+import { groupBasedRoutes } from '../router/routes';
 import { reverseObject } from '../services/reverseObject';
 
-const { user } = store.state;
+const { user } = store ? store.state : { user: undefined };
 const groups = user ? user.groups.map(({ name }) => name.toLowerCase()) : [];
```

Comment:
```suggestion
const groups = user && user.groups ? user.groups.map(({ name }) => name.toLowerCase()) : [];
```
I mean, it can be undefined, right? If that happens, `map()` will fail.

Reply:
Yes, but I did this primarily for testing purposes

The GroupGuardMixin requires the user to be logged in, hence the `user` should be in store. And if user has no groups the `user.groups` should be an empty array